packages/core/web-reporter-ui/src/components/Charts/Chart.tsx

```diff
@@ -72,10 +81,12 @@ export const Chart = ({
         width: 2,
       },
       xaxis: {
-        type: "numeric",
```

Comment:
should the type be `numeric` even if the x axis has numeric values?

This produces an interesting diff on the report component for the xaxis
<img width="710" alt="image" src="https://github.com/bamlab/flashlight/assets/4534323/168e4db4-0c68-4017-8eaa-8134cc2ee32a">

At this point, we should probably separate common options from the rest and create a graph for Evolution:

1. we can get rid of the prop `interval` in `Chart`
2. common options should be similar to:

```
const COMMON_OPTIONS = useMemo(
    () => ({
      chart: {
        id: title,
        height: 350,
        type: "line",
        animations: {
          enabled: true,
          easing: "linear",
          dynamicAnimation: {
            speed: POLLING_INTERVAL,
          },
        },
        zoom: {
          enabled: false,
        },
      },
      title: {
        text: title,
        align: "left",
        style: {
          color: "#FFFFFF",
          fontSize: "24px",
          fontFamily: "Inter, sans-serif",
          fontWeight: 500,
        },
      },
      dataLabels: {
        enabled: false,
      },
      stroke: {
        curve: "smooth",
        width: 2,
      },
      xaxis: {
        labels: { style: { colors: "#FFFFFF99" } },
      },
      yaxis: {
        min: 0,
        max: maxValue,
        labels: { style: { colors: "#FFFFFF99" } },
      },
      colors,
      legend: {
        labels: {
          colors: "#FFFFFF99",
        },
      },
      grid: {
        borderColor: "#FFFFFF33",
        strokeDashArray: 3,
      },
    }),
    [colors, maxValue, title]
  );
  ```
  
 3. use it in `EvolutionChart` moved inside the cloud project + in `ReportChart`